modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -13,6 +18,8 @@ Use the following procedure to manually add {ibm-z-name} agents with {op-system-
 . Boot your {op-system-base} KVM machine.
 
 . To deploy the virtual server, run the `virt-install` command with the following parameters:
+
+ifdef::pxe-boot[]
```

Comment:
```suggestion
ifndef::pxe-boot[]
```

Reply:
@SNiemann15 I think this ifdef logic needs to stay the way it is, because right now it ensures that lines 26-47 are only displayed in the PXE boot procedure and lines 54-68 are only displayed in the regular Agent install procedure. To my understanding that's what is intended by this PR, please see the docs preview of the PR to verify that the rendered doc looks correct to you.

Reply:
True I think I was confused by the double output and realized later what the reason is 👍